### PR TITLE
[TEST MERGE] Decastation

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

DecaStation, a small station designed to facilitate your autism forting needs without being cumbersome to work with. DecaStation is intentionally low maintenance, using a solar and turbine for power, instead of the alternatives such as a supermatter engine which is prone to destroying half the station if fucked up. Most areas which experience heavy traffic such as medbay and the production lab are given toggleable shutters to allow free passage if needed.

## Why It's Good For The Game

DecaStation is perfect for low-pop servers like ours.

## Changelog
:cl:
add: DecaStation
/:cl:

### CODE REVIEW

**Comment by jbox144** [comment]
how do I fix the conflict?

**Comment by MCterra10** [comment]
Going ahead and adding to the testmerge list.

File: austation/code/game/decastation.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	info = "Since you guys and atmos both do burns for tritium, I've decided to make it so you now share the turbine.\
+	 Go use it to make your tritium, or don't, but it ain't my problem.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/structure/fluff/deca/tcomm
+	name = "Tania's Solution"
+	desc = "A rather messy assembly of various telecommunication components to substitute a dedicated telecommunications mainframe. By the looks of things, it's finally given up."
+	icon = 'icons/obj/machines/telecomms.dmi'
+	icon_state = "comm_server_o_off"
```
Comment (MCterra10) [inline austation/code/game/decastation.dm:50]: @TheFakeElon 
what's your opinion on mapped character references?
I'm against it personally

Comment (TheFakeElon) [reply austation/code/game/decastation.dm:50]: >no doris mention

remove it

**Review by MCterra10 — CHANGES_REQUESTED** [review]
remove character references pls and ty

**Comment by jbox144** [comment]
gonna revise this map more

**Pull request closed without merging**